src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+const succesDiv = document.createElement('div');
+const errorDiv = document.createElement('div');
+
+succesDiv.classList.add('message');
+errorDiv.classList.add('message', 'error-message');
+
+const succesMessage = document.createTextNode('Promise was resolved!');
+const errorMessage = document.createTextNode('Promise was rejected!');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = Promise.reject;
```

Comment:
You should reject your promise in 3 seconds. You need to use syntax like above `new Promise((resolve, reject) => {})` but just use `setTimeout` inside and reject only after 3 seconds has passed.